pootle/settings/50-project.conf

```diff
@@ -38,9 +38,7 @@ MIDDLEWARE_CLASSES = [
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            working_path(os.path.join('templates')),
-        ],
+        'DIRS': [],
```

Comment:
@unho, @dwaynebailey, I'll remove `DIRS` since default value is `[]` in any case.